### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Tresor11**
This is what we have done so far concerning our second project :relaxed:

### CODE REVIEW

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @Datagirlcmr and @Tresor11  🌟

**Great work on this project, we just have to fix the following before the final approval ⬇️** 

1. Fix all the **stickler errors** on your pull request.

2. Make the **README** file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

**Comment by Tresor11** [comment]
**we have fixed all the error and updated our  README** :smile:

**Comment by shshamim63** [comment]
## :tada:   <em>Status Approved!</em> :tada: 
### Hi @Tresor11  and @Datagirlcmr
 #### [Optional] (I think it will improve the view a little bit more.)
1. Can you increase the gap between password label and password input field?
2. Can you reduce the height of button a little bit?
### Congratulation, You have completed this project successfully. :clap:
#### This is from me :+1:. You can merge it now in master :tada: . Best of luck for your next project

#### Happy Coding :herb:
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

**Pull request merged**
